src/background.ts

```diff
@@ -104,9 +108,40 @@ app.on('ready', async () => {
       console.error('Vue Devtools failed to install:', e.toString())
     }
   }
-  createFirstWindow()
+  const slice = platformInfo.isDevelopment ? 2 : 1
+  const parsedArgs = yargs(process.argv.slice(slice))
+  console.log("got args", parsedArgs)
+  const options = parsedArgs._.map((url: string) => ({ url }))
+  const settings = await initBasics()
+  
+  if (options.length > 0) {
+
+    await Promise.all(options.map((option) => buildWindow(settings, option)))
+  } else {
+    if (getActiveWindows().length === 0) {
+      const settings = await initBasics()
+      await buildWindow(settings)
+    }
+  }
 })
 
+// Open a connection from a file (e.g. ./sqlite.db)
+app.on('open-file', async (event, file) => {
+  event.preventDefault();
+  console.log('opening file')
+  const settings = await initBasics()
+  
+  await buildWindow(settings, { url: file })
```

Comment:
maybe we can do something like this?
```suggestion
  await buildWindow(settings, { url: `file://${encodeURI(file)}` })
```

Reply:
that only works for mac unfortunately. I think we have enough of a solution